tadrep/utils.py

```diff
@@ -82,14 +81,25 @@ def parse_arguments():
-    arg_group_plot.add_argument('--label_wrapping', action='store', default=7,  help='Number of characters per contig track')
-    arg_group_plot.add_argument('--plasmid_head_modifier', action='store', default=100,  help='Modifier of plasmid head length')
-    arg_group_plot.add_argument('--contig_head_modifier', action='store' ,default=200,  help='Modifier of contig head length')
+    arg_group_plot = visualization_parser.add_argument_group('Style')
+    arg_group_plot.add_argument('--plotstyle', action='store', default='box', dest='plotstyle', choices=['bigarrow', 'arrow', 'bigbox', 'box', 'bigrbox', 'rbox'], help='Contig representation in plot (default = "box")')
+    arg_group_plot.add_argument('--labelcolor', action='store', default='black', dest='labelcolor', help='Contig label color (default = "black")')
+    arg_group_plot.add_argument('--linewidth', action='store', default=0.0, type=float, dest='linewidth', help='Contig edge linewidth (default = 0)')
+    arg_group_plot.add_argument('--arrow-shaft-ratio', action='store', default=0.5, type=float, dest='arrow_shaft_ratio', help='Size ratio between arrow head and shaft (default = 0.5)')
+    arg_group_plot.add_argument('--size-ratio', action='store', default=1.0, type=float, dest='size_ratio', help='Contig size ratio to track (default = 1.0)')
+
+    arg_group_label = visualization_parser.add_argument_group('Label')
+    arg_group_label.add_argument('--labelsize', action='store', default=15, type=int, dest='labelsize', help='Contig label size (default = 15)')
+    arg_group_label.add_argument('--labelrotation', action='store', default=45, type=int, dest='labelrotation', help='Contig label rotation (default = 45)')
+    arg_group_label.add_argument('--labelhpos', action='store', default='center', dest='labelhpos', choices=['left', 'center', 'right'], help='Contig label horizontal position (default = "center")')
+    arg_group_label.add_argument('--labelha', action='store', default='left', dest='labelha', choices=['left', 'center', 'right'], help='Contig label horizontal alignment (default = "left")')
+
+    arg_group_gradient = visualization_parser.add_argument_group('Gradient')
+    arg_group_gradient.add_argument('--interval-start', action='store', type=float, default=80, metavar='[0-100]', dest='interval_start', help='Percentage where gradient should stop (default = 80%%)')
+    arg_group_gradient.add_argument('--number-of-intervals', action='store', type=int, default=10, choices=range(1, 101), metavar='[1-100]', dest='interval_number', help='Number of gradient intervals (default = 10)')
+
+    arg_group_omit = visualization_parser.add_argument_group('Omit')
+    arg_group_omit.add_argument('--omit_ratio', action='store', default=1, type=int, choices=range(0, 101), metavar='[0-100]', dest='omit_ratio', help='Omit contigs shorter than X percent of plasmid length from plot  (default = 1%%)')
```

Comment:
Harmonize parameter style to kebap case, _e.g._ `--important-foo`.
Currently, all possible versions (`--importantfoo`, `--important_foo` and `--important-foo`) are used.